index.md

```diff
@@ -1,213 +1,270 @@
+                <div class="homepage__key_point_title">Craft beautiful UIs</div>
+
+                <p>
+                Delight your users and make your brand stand out with
+                rich motion, smooth scrolling, and
+                beautiful customizable components.
+                </p>
+            </div>
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Used by Google</div>
+
+                <p>
+                Flutter is used by Google, works with Firebase
+                and other mobile app SDKs,
+                and is open source.
+                </p>
+            </div>
+
+        </div>
+    </section>
+
+    <section class="homepage__hot_reload">
+        <h1>Fast development</h1>
+
+        <p>
+            Flutter's <em>hot reload</em> helps you quickly
+            and easily experiment, build UIs, add features, and fix
+            bug faster. Experience sub-second reload times,
+            without losing state, on
+            emulators, simulators, and hardware for iOS
+            and Android.
+        </p>
+
+        <div class="hot-reload-gif-container">
+            <img src="/images/intellij/hot-reload.gif" class="hot-reload-gif" alt="Make a change in your code, and your app is changed instantly.">
+        </div>
+    </section>
+
+    <section class="homepage__beautiful_uis">
+        <h1>Expressive, beautiful UIs</h1>
+
+        <p>
+        Delight your users with Flutter's built-in
+        beautiful Material Design and
+        Cupertino (iOS-flavor) widgets, rich motion APIs,
+        smooth natural scrolling, and platform awareness.
+        </p>
+
+        <div class="screenshot-list">
+            <img class="screenshot" src="/images/homepage/screenshot-1.png" width="270" height="480" alt="Brand-first shopping design">
+            <img class="screenshot" src="/images/homepage/screenshot-2.png" width="270" height="480" alt="Fitness app design">
+            <img class="screenshot" src="/images/homepage/screenshot-3.png" width="270" height="480" alt="Contact app design">
+            <img class="screenshot" src="/images/homepage/ios-friendlychat.png" width="270" height="480" alt="iOS chat app design">
+        </div>
+    </section>
+
+    <section class="homepage__hot_reload">
+        <h1>Modern, reactive framework</h1>
+
+        <p>
+        Easily compose your UI with Flutter's
+        modern functional-reactive framework and
+        rich set of platform, layout, and foundation widgets.
+        Solve your tough UI challenges with
+        powerful and flexible APIs for 2D, animation, gestures,
+        effects, and more.
+        </p>
+
+{% prettify dart %}
+class MyMessage extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return new Center(
+      child: new Text('Hello, world!'),
+    );
+  }
+}
+{% endprettify %}
+
+        <p>
+        Browse the <a href="/widgets/">widget catalog</a>
```

Comment:
It might be more useful to have the link to the Widget Catalog in the "Expressive, beautiful UIs" section.